src/execution/sema/sema_builtin.cpp

```diff
@@ -2201,6 +2201,23 @@ void Sema::CheckBuiltinParamCall(ast::CallExpr *call, ast::Builtin builtin) {
   call->SetType(ast::BuiltinType::Get(GetContext(), sql_type));
 }
 
+void Sema::CheckBuiltinStringCall(ast::CallExpr *call, ast::Builtin builtin) {
+  ast::BuiltinType::Kind sql_type;
+  switch (builtin) {
+    case ast::Builtin::Lower: {
+      if (!CheckArgCount(call, 1)) {
+        return;
```

Comment:
I feel we should call ReportIncorrectCallArg to report that it was called without parameters. Otherwise tpcc just crashes.

Reply:
Over here, we're not complaining about incorrect argument. Rather we are complaining about the wrong number of arguments right. It seems like on such instances like over here
https://github.com/cmu-db/terrier/blob/c3061172ee6037bbad82bfdc94c9ca9fa5a3d95f/src/execution/sema/sema_builtin.cpp#L1754

We just return as is here.